docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+# Getting Started
```

Comment:
I think we can remove this H1

Reply:
Just remove the line altogether or the `#`?

Reply:
The whole line (as in the latest version of this page)